airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala

```diff
@@ -539,7 +539,7 @@ object LogicalPlan {
           query.outputAttributes.zip(aliases).map { case (in, alias) =>
             SingleColumn(
               in,
-              Some(alias),
+              Some(alias.sqlExpr),
```

Comment:
If the alias is Identifier, using `Identifier.value` would be better to remove double quotation. 

`Identifier.value` method for case class may not work in Scala 2.13/3, so renaming value method to such as `.idString` might be necessary.